src/matrix-utils.ts

```diff
@@ -79,13 +80,27 @@ export async function initClient(
     storeOpts.cryptoStore = new MemoryCryptoStore();
   }
 
+  // XXX: we read from the URL search params in RoomPage too:
+  // it would be much better to read them in one place and pass
+  // the values around, but we initialise the matrix client in
+  // many different places so we'd have to pass it into all of
+  // them.
+  const params = new URLSearchParams(window.location.search);
+  // disable e2e only if enableE2e=false is given
+  const enableE2e = params.get("enableE2e") !== "false";
```

Comment:
What happens if someone tries to join an encrypted call without encryption?

Reply:
There aren't really 'encrypted calls' as such: clients with this flag set to false will still decrypt any encrypted signalling they receive, so (I think) it should all still work. However, this is very deliberately hidden in an undocumented URL param, so anyone playing with it gets to keep both pieces if it breaks. :)

Reply:
Ah, I see